croissant-rdf/scripts/networkx.py

```diff
@@ -0,0 +1,75 @@
+import rdflib
```

Comment:
Maybe add a description about the scripts' utility or how to use them.

Reply:
Note: probably should just remove and use the notebook only